scripts/main.py

```diff
@@ -272,6 +273,7 @@ def parse_arguments():
 
     parser = argparse.ArgumentParser(description='Process arguments.')
     parser.add_argument('--continuous', action='store_true', help='Enable Continuous Mode')
+    parser.add_argument('--riskAvoidance', action='store_true', help='Enable Risk Avoidance (Hybrid) Mode')
```

Comment:
I'm personally not into lower camel case flags. I'd prefer `--risk-avoidance`.

Reply:
I don't have an opinion on this, whatever is agreed upon is fine by me.